assets/badge.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="102" height="20">
+  <script/>
+  <linearGradient id="a" x2="0" y2="100%">
+    <stop offset="0" stop-color="#bbb" stop-opacity=".1"/>
+    <stop offset="1" stop-opacity=".1"/>
+  </linearGradient>
+  <rect rx="3" width="102" height="20" fill="#555"/>
+  <rect rx="3" x="64" width="38" height="20" fill="#db654f"/>
+  <path fill="#db654f" d="M64 0h4v20h-4z"/>
+  <rect rx="3" width="102" height="20" fill="url(#a)"/>
+  <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
+    <text x="32" y="15" fill="#010101" fill-opacity=".3">document</text>
+    <text x="32" y="14">document</text>
+    <text x="82.5" y="15" fill="#010101" fill-opacity=".3">32%</text>
+    <text x="82.5" y="14">32%</text>
```

Comment:
I think so, I've fetched your branch and rebased on latest master... But don't bother, will update it soon anyway :)